3761/1/page_parser.rb

```diff
@@ -0,0 +1,59 @@
+class PageParser
+  attr_reader :page
+  attr_reader :parser_data
+
+  PARSER_DATA = { used_by: 0, watch: 0, star: 0, fork: 0, contributors: 0,
+                  issues: 0 }.freeze
+
+  def initialize(page)
+    @page = page
+    @parser_data = {}
+  end
+
+  def parse_data_without_used
+    find_first_line
+    find_second_line
+    find_third_line
+    @parser_data
+  end
+
+  def parse_used_by
+    count = page.at('.Box-header a').text.strip.tr(',', '_').to_i
+    count
+  end
+
+  private
+
+  def add_in_data(par, count)
+    count = count.text.strip.tr(',', '_').to_i
+    par = par.text.strip.downcase.to_sym
+    @parser_data[par] = count if PARSER_DATA.key?(par)
```

Comment:
Why you make this check?

Reply:
because some [par, count] are useless for me